src/components/calcite-shell/calcite-shell.tsx

```diff
@@ -79,9 +79,11 @@ export class CalciteShell {
       <div class={mainClasses}>
         <slot name={SLOTS.primaryPanel} />
         {this.renderContent()}
-        <slot name={SLOTS.contextualPanel} />
```

Comment:
@asangma can you revert these changes? We need to keep the `tipManager` until we can do breaking changes. Also, I think contextual panel should be tabbed to before `bottomPanel` right?

Reply:
re: tipManager
I can revert, but is anybody using that at the moment?

Reply:
re: contextual panel before centerRow (previously bottomPanel)
If I do that, this happens:
![image](https://user-images.githubusercontent.com/12503298/80770025-e0bf0800-8b03-11ea-80d6-6a0d02170c82.png)

I could use `display: grid` on `calcite-shell` > `.main`, but I would like to do that in a different PR maybe?

Reply:
Hmmm...I could use flex order too I guess...?

Reply:
Like a dis.
```
slot[name="primary-panel"]::slotted(*) {
  order: 1;
}
slot[name="center-row"]::slotted(*) {
  order: 2;
}
slot[name="contextual-panel"]::slotted(*) {
  order: 3;
}
```
Thoughts?

Reply:
@driskull Made some updates. Lemme know what you think.

Reply:
> I can revert, but is anybody using that at the moment?

Lets keep it because if we remove then its a breaking change.

> Hmmm...I could use flex order too I guess...?

We can move the node up if you think the center should come before contextual 👍 

Lets just keep the tipManager slot until we do a larger breaking change

Reply:
Sounds good. Thanks man.

Reply:
@driskull TipManager slot is back. Nodes reordered. 👍🏽